README.md

```diff
@@ -1,32 +1,28 @@
-## Go Ethereum
+## Go Quai
 
-Official Golang implementation of the Ethereum protocol.
+Official Golang implementation of the Quai protocol.
 
 [![API Reference](
 https://camo.githubusercontent.com/915b7be44ada53c290eb157634330494ebe3e30a/68747470733a2f2f676f646f632e6f72672f6769746875622e636f6d2f676f6c616e672f6764646f3f7374617475732e737667
-)](https://pkg.go.dev/github.com/ethereum/go-ethereum?tab=doc)
-[![Go Report Card](https://goreportcard.com/badge/github.com/ethereum/go-ethereum)](https://goreportcard.com/report/github.com/ethereum/go-ethereum)
+)](https://pkg.go.dev/github.com/dominant-strategies/go-quai/common)
+[![Go Report Card](https://goreportcard.com/badge/github.com/spruce-solutions/go-quai)](https://goreportcard.com/report/github.com/spruce-solutions/go-quai)
+<!--
 [![Travis](https://travis-ci.com/ethereum/go-ethereum.svg?branch=master)](https://travis-ci.com/ethereum/go-ethereum)
-[![Discord](https://img.shields.io/badge/discord-join%20chat-blue.svg)](https://discord.gg/nthXNEv)
+-->
+[![Discord](https://img.shields.io/badge/discord-join%20chat-blue.svg)](https://discord.gg/Nd8JhaENvU)
 
 Automated builds are available for stable releases and the unstable master branch. Binary
 archives are published at https://geth.ethereum.org/downloads/.
 
-## Running the linter
-
-Before every commit  the linter has to be run for code sanity purposes along with other reasons.
-Run the command below to download and run the `golangci-lint`.
-```make lint```
-
 ## Building the source
 
 For prerequisites and detailed build instructions please read the [Installation Instructions](https://geth.ethereum.org/docs/install-and-build/installing-geth).
 
-Building `geth` requires both a Go (version 1.14 or later) and a C compiler. You can install
+Building `go-quai` requires both a Go (version 1.14 or later) and a C compiler. You can install
```

Comment:
change to 1.19

Reply:
Looks like we also need to update the docs here:
https://docs.quai.network/develop/installation#prerequisites

Reply:
New docs are now updated per Judi